[ci]  Fixes #2 Added file travis and Makefile

m4yk3ldev opened the pull request.

Commit:
[travis]  refs #2 Added file travis and makefile

Commit:
[Makefile] refs #2 Change in Makefile

Commit:
refs #2 Configure travis

olemis commented on src/Lykke.Service.Skycoin.Sign/Startup.cs:
```diff
@@ -31,7 +31,7 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
                 {
                     logs.AzureTableName = "Skycoin.SignLog";
-                    logs.AzureTableConnectionStringResolver = settings => settings.Skycoin.SignService.Db.LogsConnString;
+                    logs.AzureTableConnectionStringResolver = settings => settings.SkycoinSignService.Db.LogsConnString;
```
Revert this change and update namespaces accordingly.

m4yk3ldev replied:
@olemis  the names of functions can not have a point, they are limited.

https://github.com/simelotech/exchange_sign/pull/3/commits/4a7e89ec58927a0ccdb63f2109b2ccbf705b23ab#diff-1c50f3c7059db3cf6563050da7caac36

olemis committed:
[ci] refs #2 - Expected make targets (WiP). Implement make help

Commit:
refs #3 Changes to be committed:
	modified:   .travis.yml

Commit:
refs #3 Changes to be committed:
	modified:   src/Lykke.Service.Skycoin.Sign/Settings/AppSettings.cs
	modified:   src/Lykke.Service.Skycoin.Sign/Settings/Skycoin.SignSettings.cs
	modified:   src/Lykke.Service.Skycoin.Sign/Startup.cs
